app/src/main/res/drawable/ic_code_vector.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillColor="#FFFFFF"
+        android:pathData="M8.7 15.9L4.8 12l3.9-3.9c0.39-0.39 0.39-1.01 0-1.4-0.39-0.39-1.01-0.39-1.4 0l-4.59 4.59c-0.39 0.39-0.39 1.02 0 1.41l4.59 4.6c0.39 0.39 1.01 0.39 1.4 0 0.39-0.39 0.39-1.01 0-1.4zm6.6 0l3.9-3.9-3.9-3.9c-0.39-0.39-0.39-1.01 0-1.4 0.39-0.39 1.01-0.39 1.4 0l4.59 4.59c0.39 0.39 0.39 1.02 0 1.41l-4.59 4.6c-0.39 0.39-1.01 0.39-1.4 0-0.39-0.39-0.39-1.01 0-1.4z" />
+</vector>
```

Comment:
Now that I saw it here https://fonts.google.com/icons?icon.query=swap+horiz&icon.platform=android (the one w/o the circle) can you use that Icon instead? I think it is a bit more intuitive of what it does. 🙏